index.html

```diff
@@ -5,11 +5,152 @@
+            <img src="images/facebook.png" alt="facebook">
+            <img class="twitter-img" src="images/twitter.png" alt="twitter">
+            <a class="a-header-social" href="">English</a>
+            <a class="a-header-social" href="">My Page</a>
+            <a class="a-header-social" href="">Logout</a>
+        </div>
+        <header class="header">
+            <div class="logo-div">
+                <img src="images/logo.png" alt="logo-header" class="logo-img">
+            </div>
+            <section class="a-header-div" id="desk-buttons">
+                <a href="javascript:void(0)" id="clsebtn" class="closebtn">X</a>
+                <div class="header-ul">
+                    <a class="a-buttons a-home" href="index.html">Home</a>
+                    <a class="a-buttons" href="about.html">About</a>
+                    <a class="a-buttons" href="#">Program</a>
+                    <a class="a-buttons" href="#">Join</a>
+                    <a class="a-buttons" href="#">Sponsor</a>
+                    <a class="a-buttons" href="#">News</a>
+                    <button class="ccc-button">CC Campaign</button>
+                </div>
+            </section>
+        </header>
+    </div>
+
+<!-- -----------------------COVER--------------------------------- -->
+
+    <div class="intro-mobile">
+        <div class="hamburguer">
+            <span id="open-one" class="toggle-nav">&#9776;</span>
+        </div>
+        <div class="main-title">
+            <p class="title-paragraph">Hello Rockers in the world!</p>
+            <h1 class="title">VPN LIVE IN CONCERT WORLD TOUR 2022</h1>
+        </div>
+        <div class="paragraph">
+            <p class="paragraph-text">Join us for live music, food trucks and much more fun for all the family. Featuring one of the most popular bands in Quito, Ecuador VPN!. See you there and lets rock!</p>
+        </div>
+        <div class="date-place">
+            <p class="date">2022.02.10(THU) ~ 11(FRI)</p>
+            <p class="place">@Coliseo General Rumiñahui, Quito EC</p>
+        </div>
+    </div>
+    <div class="intro-desk">
+        <div class="main-title">
+            <p class="title-paragraph">Hello Rockers in the world!</p>
+            <h1 class="title">VPN LIVE IN CONCERT WORLD TOUR 2022</h1>
+        </div>
+        <div class="paragraph">
+            <p class="paragraph-text">Join us for live music, food trucks and much more fun for all the family. Featuring one of the most popular bands in Quito, Ecuador VPN!. See you there and lets rock!</p>
+        </div>
+        <div class="date-place">
+            <p class="date">2022.02.10(THU) ~ 11(FRI)</p>
+            <p class="place">@Coliseo General Rumiñahui, Quito EC</p>
+        </div>
+    </div>
+
+<!-- -----------------------PROGRAM--------------------------------- --> 
+
+    <div class="main-program">
+        <div class="main-p-title">
+            <h2 class="program-title">Main Program</h2>
```

Comment:
- MOBILE VERSION

- [ ] Please adjust this main section as per the design, the text on the right are not aligned on the same line, see the image below compared to the original one and make it look similar, please
- [ ] Take a look at your icons, they are different from those in the original by the color, please use some with white and orange color to make your app look more beautiful.
> | | Your design | | Figma       | |
> |-|------------  |-|----------|-|
> | | <img width="253" alt="align" src="https://user-images.githubusercontent.com/79058364/157560813-7a530bf5-378a-4e66-b3c1-188cc172f26a.png"> | | <img width="388" alt="figmaaa" src="https://user-images.githubusercontent.com/79058364/157560835-d82cff98-ab43-4323-9a06-cf7f02ad9798.png"> | |

- [ ] Still on the mobile version, referring to this image below, by default your page should render only 2 feature speaker members, then when the user clicks the `more` button, he should be able to see the remaining 4 members, please implement the see more and see less functionality.

<img width="389" alt="seemore" src="https://user-images.githubusercontent.com/79058364/157563961-b780eafe-87ba-4279-bd4b-6609a0822d47.png">

Reply:
#  Hey Juanse

- [ ] Please consider this change as Optional the MORE and LESS functionality is not required, I am sorry to mark it so since it is not meant to be one, Happy coding, enjoy the great work 💪🏻 .

Reply:
Hi dear Ben, it's okay for me but, please take into consideration that I was already working on this instead of working on the others requested changes because this is the hardest part from my point of view.

So maybe it is a little bit frustrating (for me and for future students) being asked to do something that is not required and expends a lot of time to do.